doc/source/reference/c-api/array.rst

```diff
@@ -3443,6 +3443,11 @@ Other constants
     variable is defined to guarantee the numpy/arrayobject.h header is
     being used).
 
+.. c:macro:: NPY_FEATURE_VERSION
+
+    The current version of the API. :c:data:`NPY_FEATURE_VERSION` changes
+    whenever the API changes (e.g. a function is added).
```

Comment:
The comment in the source is
```
/*
 * Minor API version.  This number is increased whenever a change is
 * made to the C-API -- whether it breaks binary compatibility or not.
 * Some changes, such as adding a function pointer to the end of the
 * function table, can be made without breaking binary compatibility.
 * In this case, only the NPY_FEATURE_VERSION (*not* NPY_VERSION)
 * would be increased.  Whenever binary compatibility is broken, both
 * NPY_VERSION and NPY_FEATURE_VERSION should be increased.
 */
```

It might be good to capture a little more of this.

Reply:
Actually, I think a better idea would be to move these two `macro`s up to the https://numpy.org/doc/stable/reference/c-api/array.html?highlight=npy_feature_version#checking-the-api-version section higher in this file

Reply:
Moved these macros.